app/assets/javascripts/discourse/templates/modal/upload-customization.hbs

```diff
@@ -0,0 +1,8 @@
+<form {{action "dummy" on="submit"}}>
+  <div class='modal-body'>
+    {{json-file-uploader value=customizationFile extension=".dcstylejson"}}
```

Comment:
really not a fan of dcstylejson its a real weird extension.

either 

dcstyle

or 

dcstyle.json if we must